features/alphabetSearch/dataTables.alphabetSearch.js

```diff
@@ -184,9 +197,6 @@ $.fn.dataTable.AlphabetSearch = function ( context ) {
 	};
 };
 
-$.fn.DataTable.AlphabetSearch = $.fn.dataTable.AlphabetSearch;
```

Comment:
I am not sure why this was there at all. Removed it as I believe it's some kind of mistake.

Reply:
Actually it is needed - `$.fn.dataTable` and `$.fn.DataTable` are actually different objects - but for ease we tend to make what is available on one, available on the other.

Reply:
Fixed